### PR TITLE
Add jiggle physics, Add Rubber Sniper

### PR DESCRIPTION
**Pull request by Fueredoriku**
jiggleMeshSniperFinal3
- Add Rubber Sniper, a new extension
   - Deals a lot of damage, and has a very long range
   - Jiggles on movement, making hitting players very hard unless moving slowly and carefully
- Add jiggle mesh, for general jiggling needs
jiggleMeshSimple
- Add very basic general Jiggle bone for transforms

### CODE REVIEW

File: Assets/Scripts/Utils/WobbleMesh.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WobbleMesh : MonoBehaviour
+{
+    [SerializeField]
+    private Material jiggleMat;
```
Comment (toberge) [inline Assets/Scripts/Utils/WobbleMesh.cs:8]: Abbreviation 😤

File: Assets/Scripts/Utils/WobbleMesh.cs
Comment (toberge) [inline Assets/Scripts/Utils/WobbleMesh.cs:1]: Why wobble when jiggle

**Review by toberge — APPROVED** [review]
Is probably fine, haven't tested, just got two smol bits of backfeeding

File: Assets/Scripts/Utils/WobbleMesh.cs
Comment (Fueredoriku) [reply Assets/Scripts/Utils/WobbleMesh.cs:1]: This is a wobble script, it slopes towards previous positions and slowly returns back to the right position without oscillating (no jiggling). Like the general jiggleMesh it is currently unused, but is potentially a nice tool to have in the future 😎

File: Assets/Scripts/Utils/WobbleMesh.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/Utils/WobbleMesh.cs:8]: Oppsie, forgot that one

File: Assets/Scripts/Utils/WobbleMesh.cs
Comment (toberge) [reply Assets/Scripts/Utils/WobbleMesh.cs:1]: Oh okay, I'm still fuzzy on the wobble-jiggle dichotomy

**Pull request merged**